openquakeplatform/pla_common/management/commands/add_documents_prod.py

```diff
@@ -0,0 +1,578 @@
+import os
+import json
+from django.core.management.base import BaseCommand
+from geonode.documents.models import Document
+from geonode.layers.models import Layer, Style, Attribute
+from geonode.maps.models import Map, MapLayer
+from geonode.base.models import TopicCategory, Region, License
+from geonode.base.models import SpatialRepresentationType
+from geonode.base.models import HierarchicalKeyword
+from geonode.base.models import ResourceBase, TaggedContentItem
+from django.contrib.auth import get_user_model
+from django.contrib.contenttypes.models import ContentType
+from agon_ratings.models import OverallRating
+from decimal import Decimal
+
+
+def base_attrs(base):
+    base_new = {}
+    base_new.update(base)
+    # base_new['thumbnail_url'] = base['thumbnail']
```

Comment:
add a comment about it or remove it

Reply:
Fixed here: https://github.com/gem/oq-platform2/pull/66/commits/21b856a13ff089decb911df09b289bf5e3963e50